web/src/api/http.ts

```diff
@@ -55,7 +55,7 @@ const put = (url: string, data: object) => http.put(url, data);
  * @param url - endpoint URL
  * @param data - request payload
  */
-const post = (url: string, data: object) => http.post(url, data);
+const post = (url: string, data?: object) => http.post(url, data);
```

Comment:
hmm, for both changes, does it make sense to have patch and post that does not provide data? It looks like bug for me.

Reply:
after seeing below usage I would say that post change makes sense...but still for patch I kind of expect instructions how to modify source and below usage is at least confusing for me.

Reply:
Which verbs use for which actions / request will be for sure a topic for discussions. I personally have to revisit all the spec for refreshing my mind about HTTP verbs and the best usage of them. But as per RFC, https://www.rfc-editor.org/rfc/rfc5789, patch does not require a payload. It's optional.

As you pasted in another comment, `A PATCH request is considered a set of instructions on how to modify a resource`. In the network case, we're using PATCH for request a `connect` or `disconnect` _instructions_ (aka actions) which tells the server how to modify the resource: connection or disconnecting it.